### DISCUSSION TITLE
可否添加洗脑循环自动开启

### DISCUSSION THREAD

**Discussion by StrangerBWL** [the1812/Bilibili-Evolved#1857] [功能建议 / Ideas]
1，功能
打开视频洗脑循环能自动开启

2.为什么
因为我单纯比较喜欢在B站音乐区听音乐或者循环视频...

3.建议
如果可以，能否在收藏夹里添加自己可以自定义哪一个视频能自动开启洗脑循启

4.Thank

**Comment by Geeyun-JY3** [comment]
重复：#987
